### ISSUE TITLE
Add functionality to integrate more modules with civilian hostility

### ISSUE THREAD

**Issue by tupolov** [ALiVEOS/ALiVE.OS#425]
These ideas make use of the new Laws of War DLC concept and assets.

**Decreasing Civilian Hostility:**
IED disarm
Building a fortification / well / something
Placing an AI group in the village on patrol
Clearing CQB positions
Destroying roadblocks (needs a destruct/delete mechanic)
Positive civilian interaction (where is Spyder's addon!?)

_New:_
Picking up anti-personnel mines
Removing UXO
IDAP resupply of assets in village/town
IDAP personnel placed in village/town
Establish IDAP medical tents.
Dropping friendly leaflets
Displays of force (low flight over civ areas with enemy presence)

**Increasing civ hostility:**
Killing civs
Destroying civ buildings
Negative civ interaction
Insurgent installation established in area
IED / Roadblocks

_New:_
Removal of IDAP assets
Destruction of medical tents
Killing IDAP folks
Dropping cluster bombs near civs
Enemy OPCOM dropping leaflets
As for Civ behaviour - the civ pop system today very much supports managing their behaviour, so its not a stretch to do the following:

**Civ hostility low:**
Crowds / Individuals exit buildings to welcome player groups as they approach town, with anims to wave, salute - might need some ambient crowd sounds such as cheering, clapping etc.
Individuals offer intel on enemy.
Civs spend more time outside
Civs form groups outside conversing
Civs request players to attack enemy positions/groups etc

**Civ hostility high:**
Civs throwing stones.
Civs jeering.
Civs staying in houses
Civs avoiding friendlies
No intel
Civs share player positions with enemy / call for enemy to attack

Just need a developer to pick this up! ;)

**Comment by tupolov** [comment]
Example code

```
[_pos,_sides, 20] call ALiVE_fnc_updateSectorHostility;
| [_pos,_allSides - _sides, -20] call ALiVE_fnc_updateSectorHostility;
```

**Comment by HeroesandvillainsOS** [comment]
YES! Just commenting so I'll get an email anytime something lovely happens. :)

**Comment by afreeland88** [comment]
Did these ever get implemented ? the IDAP stuff is exactly whats missing from a hearts and mind campaign would be great to be able to drop of food and water from the laws of war dlc and it have an effect on the population

**Comment by arismagicgr** [comment]
It's been !**7**! years since this issue was created but as ALiVE's Insurgency System is one of the best thi fa that has happened to ArmA 3 (possibly with ACE3 and ACRE2), I believe that such a "_framework_" would be great... **BUT**... With the functions that change the Civ Hostility to a sector already there, I would say that almost any _scripted_ mission can be used here and actually have an impact on Civ Hostility... It just may need some minor adjustments to implement conditions that will change the Civ Hostility according to the outcome (e.g. if players are tasked to escort a doctor to a remote village, if the doctor makes it and comes back safe, +20 for players' side, if the doctor makes it to the village, do its job but does not make it back, no earn, if the doctor does not make it there (for any reason, abandon assignment or doctor dies), -20 for players' side) and boom... You are good to go...

I may miss things here since scripting, programming and codes are not my strong point but... That's the general idea....
